OpenRA.Game/Settings.cs

```diff
@@ -241,6 +241,19 @@ public Hotkey GetSupportPowerHotkey(int index)
 		}
 	}
 
+	public class IrcSettings
+	{
+		public string Hostname = "chat.freenode.net";
```

Comment:
I want to go with `irc.openra.net` again or if that is not possible ask http://cncnet.org.